stixcore/idb/idb.py

```diff
@@ -59,11 +61,10 @@ def get_idb_filename(self):
     def _connect_database(self):
         try:
             self.conn = sqlite3.connect(str(self.filename), check_same_thread=False)
-            #logger.info('IDB loaded from {}'.format(filename))
+            logger.info('IDB loaded from {}'.format(self.filename))
             self.cur = self.conn.cursor()
-            #self.conn.row_factory = sqlite3.Row
         except sqlite3.Error:
-            #logger.error('Failed load IDB from {}'.format(filename))
+            logger.error('Failed load IDB from {}'.format(self.filename))
```

Comment:
Probably best to raise the exception again as it not going to work without the db